src/nvim/os/env.c

```diff
@@ -390,14 +390,14 @@ static char_u *vim_version_dir(char_u *vimdir)
 
 /// If the string between "p" and "pend" ends in "name/", return "pend" minus
 /// the length of "name/".  Otherwise return "pend".
-static char_u *remove_tail(char_u *p, char_u *pend, char_u *name)
+static char *remove_tail(char *p, char *pend, char *name)
 {
-  int len = (int)STRLEN(name) + 1;
-  char_u      *newend = pend - len;
+  size_t len = STRLEN(name) + 1;
```

Comment:
You can start replacing `STRLEN` with `strlen` when the string is `char *`. `STRLEN` is just a `strlen` wrapper that casts the string from `char_u *` to `char *`.